Gopkg.toml

```diff
@@ -301,11 +300,15 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.9.1"
+  version = "kubernetes-1.10.4"
```

Comment:
Why api use `1.10.4` but kubernetes use `1.10`? I found that Kubernetes released `1.10.4` as well at https://github.com/kubernetes/kubernetes/releases/tag/v1.10.4

Reply:
OK, I have pushed again with 1.10.4 everywhere.